runtime/compiler/z/codegen/J9CodeGenerator.cpp

```diff
@@ -123,6 +123,12 @@ J9::Z::CodeGenerator::initialize()
       cg->setSupportsInlineEncodeASCII();
       }
 
+   static char *disableInlineVectorizedMismatch = feGetEnv("TR_disableInlineVectorizedMismatch");
+   if (cg->getSupportsArrayCmp() && !disableInlineVectorizedMismatch)
```

Comment:
Although functionally it wouldn't make difference, should explicitly check `disableInlineVectorizedMismatch` against NULL instead of checking it as condition.